oslo/transformers/tasks/data_bart_pretraining.py

```diff
@@ -109,35 +107,42 @@ def __call__(self, examples: List[Dict[str, Any]]) -> Dict[str, Any]:
         batch = self.tokenizer.pad(
             examples,
             return_tensors="pt",
-            pad_to_multiple_of=self.pad_to_multiple_of,
+            pad_to_multiple_of=self.local_world_size
+            if self.local_world_size > 1
+            else None,
         )
 
-        if self.pad_to_multiple_of is not None:
+        if self.local_world_size > 1:
```

Comment:
let's make some utility functions for this. 
this can make reduce redundancy. :)